### DISCUSSION TITLE
Max Payne 3 error while applying cheats

### DISCUSSION THREAD

**Discussion by dolixec** [bucanero/apollo-ps3#61] [Q&A] [answered]
Hi there.
I am using the latest version of this app. I've used it on other games and it was all a success, but when I tried to apply two cheats on Max Payne 3 I got an error message that the cheats couldn't have been applied, followed by a message that confirmed the save was changed. In game the cheats weren't applied.
Any fix for this, or is it related to a bug?

**Comment by bucanero** [comment]
This only means that the patches couldn't be applied (e.g., the file or data to be modified couldn't be found, etc.), and is usually related to the cheat codes (no issue on the app). The codes might be wrong or haven't been verified, etc.

Checking the Max Payne 3 codes, it looks like they're only valid with checkpoint save files, so those codes can't be applied to every save:

```
; Max Payne 3
; BLUS30557
;From Game Genie For PS3

:BLUS30557MP3_PROGRESSION\RAGE.SAV

; Only available on checkpoints (not the start of a chapter) in Story Mode > Continue.
[Max Pills]
....

; Must have weapons equipped. Only available on checkpoints (not the start of a chapter) in Story Mode > Continue.
[Max Ammo]
....
```

I'd suggest to check that first... if that doesn't work, you might need to ask in BSD or GameGenie forums to see if there's any info or update for those cheat codes.

**Comment by dolixec** [comment]
The thing is, those cheats are for BLUS, as far as I see. The game that I tried to apply those 2 cheats was BLES. I don't think it makes a really important difference, since the app was showing them while editing it.

**Comment by bucanero** [comment]
> The thing is, those cheats are for BLUS, as far as I see. The game that I tried to apply those 2 cheats was BLES. I don't think it makes a really important difference, since the app was showing them while editing it. 

The Euro codes are the same, and have the same notes regarding the checkpoint:

https://github.com/bucanero/apollo-ps3/blob/master/appdata/BLES00942.ps3savepatch

**Comment by bucanero** [comment, marked as answer]
btw, I was reviewing some information, and Max Payne 3 saves on Xbox and PC are heavily (probably "insanely" is more accurate) encrypted, so I assume that PS3 saves are also following the same insanity.
(the original GameGenie tool might have solved that encryption)

Anyways, since there's no save decrypter for PS3, I think you'll have a better chance using in-memory cheats with Artemis or NetCheat, than trying to modify the Max payne 3 save.

I think I'll tag these codes as "non-working" or "need decryption"

**Reply by Coioteodg** [reply]
Hi there.
I am using the latest version of this app. I've used it on other games and it was all a success, but when I tried to apply two cheats on Resident evil operation raccon city, rage, the evil with I got an error message that the cheats couldn't have been applied, followed by a message that confirmed the save was changed. In game the cheats weren't applied.
Any fix for this, or is it related to a bug?

**Comment by Coioteodg** [comment]
Hi there.
I am using the latest version of this app. I've used it on other games and it was all a success, but when I tried to apply two cheats on Residente evil operation raccon city, rage , the evil with I got an error message that the cheats couldn't have been applied, followed by a message that confirmed the save was changed. In game the cheats weren't applied.
Any fix for this, or is it related to a bug?

**Comment by bucanero** [comment]
(Many years later) 
I made a Max Payne 3 save decrypter that can handle PS3 saves:
- https://github.com/bucanero/save-decrypters/tree/master/maxpayne3-decrypter
